### DISCUSSION TITLE
使用vitest进行测试会报错

### DISCUSSION THREAD

**Discussion by liuzaijiang** [antvis/X6#4676] [General]
### 问题描述

测试文件或者测试的相关文件引入@antv/x6，则跑测试案例的时候会报错

### 重现链接

https://codesandbox.io/s/interesting-lamport-7jhr6s?file=/test/basic.test.tsx

### 重现步骤

npm run test，使用vitest进行测试

### 预期行为

不报错

### 平台

- 操作系统: [macOS, Windows, Linux, React Native ...]
- 网页浏览器: [Google Chrome, Safari, Firefox ...]
- X6 版本: [2.15.3]
- vitest:[0.34.6]

### 屏幕截图或视频（可选）

![image](https://github.com/antvis/X6/assets/26105153/df71148f-681b-4635-b31d-bb38ca1c22f4)

[a comment by an automated account is not shown]

**Comment by NewByVector** [comment]
这个看起来是 vitest 的配置问题呀。
